### DISCUSSION TITLE
Safari cannot create peaks

### DISCUSSION THREAD

**Discussion by korayrubicoda** [katspaugh/wavesurfer.js#2857] [Q&A]
I'm using this code to generate peaks.

`useEffect(() => {
    ws.current = WaveSurfer.create({ container: wavDiv.current })
    ws.current.on("ready", function () {
      const peaks = ws.current.backend.getPeaks(300, 0, 300);
      const duration = ws.current.backend.getDuration();
      setPeaks(peaks);
      setDuration(duration);
    });
  })`
It generates correctly in Chrome, but it doesn't generate peaks properly in Safari.

Safari
![image](https://github.com/katspaugh/wavesurfer.js/assets/123622521/4b419b11-3fae-4189-92c8-50b8a1892422)

Chrome
![Screenshot 2023-06-06 at 14 22 10](https://github.com/katspaugh/wavesurfer.js/assets/123622521/80c54892-0d0e-417a-bf43-b03eb8c54569)

How do i solve ?

**Comment by katspaugh** [comment]
I can’t offer support for v6 anymore, but feel free to try with the v7 beta.
